gslb/cache/controller_obj_cache.go

```diff
@@ -182,13 +183,18 @@ func (h *AviHmCache) AviHmObjCachePopulate(client *clients.AviClient, hmname ...
 			if hm.MonitorPort != nil {
 				monitorPort = *hm.MonitorPort
 			}
-			cksum := gslbutils.GetGSLBHmChecksum(*hm.Name, *hm.Type, monitorPort)
+			description := ""
+			if hm.Description != nil {
+				description = *hm.Description
+			}
+			cksum := gslbutils.GetGSLBHmChecksum(*hm.Type, monitorPort, []string{description})
```

Comment:
This can generate different checksums if the description from the GS object has different ordering of fqdns as compared to the HM object from the graph layer, isn't it? If that's the case, let's store the parsed values in the description and then create a string out of it by sorting the list. Please have a look at how we generate the checksum for GS objects by sorting the members of a GS and then creating a string out of it.